library/core/src/macros/mod.rs

```diff
@@ -831,6 +831,27 @@ pub(crate) mod builtin {
         ($fmt:expr, $($args:tt)*) => {{ /* compiler built-in */ }};
     }
 
+    /// Same as `format_args`, but can be used in some const contexts.
+    #[cfg(not(bootstrap))]
+    #[unstable(feature = "const_format_args", issue = "none")]
+    #[allow_internal_unstable(fmt_internals, const_fmt_arguments_new)]
+    #[rustc_builtin_macro]
+    #[macro_export]
+    macro_rules! const_format_args {
+        ($fmt:expr) => {{ /* compiler built-in */ }};
+        ($fmt:expr, $($args:tt)*) => {{ /* compiler built-in */ }};
+    }
+
+    /// Same as `format_args`, but can be used in some const contexts.
```

Comment:
If it is exactly the same, then why have a separate macro?

Reply:
Because we can keep const_format_args unstable for now.

Reply:
Oh, I missed the part where the panic macros use `const_format_args!` in some cases.

But still, please add a comment that this is for feature gating purposes only and should be removed once this feature becomes stable.